test/api/v3/integration/members/POST-transfer_gems.test.js

```diff
@@ -4,6 +4,14 @@ import {
 } from '../../../../helpers/api-v3-integration.helper';
 import { v4 as generateUUID } from 'uuid';
 
+function findMessage (messages, receiverId) {
+  let message = _.find(messages, (inboxMessage) => {
```

Comment:
It's a bit cleaner this way, so I agree with crookedneighbor's choice. It should have no noticeable overhead either.